src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/conf/ReadPropertiesStepExecution.java

```diff
@@ -103,4 +112,30 @@ private void addAll(Map src, Map<String, Object> dst) {
             dst.put(e.getKey() != null ? e.getKey().toString(): null, e.getValue());
         }
     }
+
+    /**
+     * Using commons collection to interpolated the values inside the properties
+     * @param properties the list of properties to be interpolated
+     * @return a new Properties object with the interpolated values
+     */
+    private Properties interpolateProperties(Properties properties) {
+        if ( properties == null)
+            return null;
+        Configuration interpolatedProp;
+        PrintStream logger = listener.getLogger();
```

Comment:
You need to rebase your master branch. The `listener` field has been removed so you are getting a compilation failure here.